packages/p2p-v2/src/components/Modals/AdConditionsModal/AdConditionsModal.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
are we not adding test cases for this modal?

Reply:
ah i didnt see it in the screenshots in the description, can ya update accordingly 👍

Reply:
its already there in the screenshot.